acceptance/Gemfile

```diff
@@ -10,7 +10,7 @@ def location_for(place, fake_version = nil)
   end
 end
 
-gem "beaker", *location_for(ENV['BEAKER_VERSION'] || "~> 2.27")
+gem "beaker", *location_for(ENV['BEAKER_VERSION'] || "~> 2.31")
```

Comment:
This PR can't be merged until there's a beaker release containing the fix for https://github.com/puppetlabs/beaker/pull/1044, and this constraint may need to be updated e.g. `2.32`.